config/overlays/kapp-config.yaml

```diff
@@ -0,0 +1,31 @@
+---
+apiVersion: kapp.k14s.io/v1alpha1
```

Comment:
I understand that some folks may use kapp to install kpack but, is this appropriate in the kpack repo?